apiary.apib

```diff
@@ -2206,6 +2206,7 @@ Accessible by users with Admin, Lead or Researcher role.
 
 + Parameters
     + Limit (integer, optional) - The number of measurements to pull from the list.
+        + Upper Limit 100
```

Comment:
Can you also add the upper limit for EP: https://dfxapiversion10.docs.apiary.io/#reference/0/measurements/list, thanks.

Reply:
This is happening from one place